CIP-1776/README.md

```diff
@@ -0,0 +1,84 @@
+License: CC-BY-4.0
+---
+
+# The Non-Tyrannical Alternative: An On-Chain Decentralized Governance Mechanism for Voltaire
+"It is dangerous to be right in matters on which the established authorities are wrong" 
+-Voltaire
+
+## Abstract
+We refute the deeply flawed [CIP-1694](https://github.com/JaredCorduan/CIPs/edit/voltaire-v1/CIP-1694/README.md) proposed by Input Output Global for an on-chain governance system and propose an alternative, more decentralized system of governance incorperating and improving upon Project Catalyst.
+
+Currently funds are dispersed from the Cardano Treasuary under an approval/disapproval voting system that exposes our system of governance to a number of critical vulnerabilities and perversely incentivized attack vectors that are routinely exploited by hostile actors. Here we propose a new method, **non-incentivized iterative block approval voting**, which is a variant of multiwinner approval voting modified to resolve the Burr dilemma.
+
+Under our proposal **any Cardano user** will be able to submit a governance action. The release of funds from the Cardano treasuary, including funds previously dispersed by Project Catalyst, will be considered a governance action. The ratification of governance actions will be the responsibility of the following parties:
+
+1. The Supreme Court (SC)
+2. Stake Pool Operators (SPOs)
+3. Delegated Representatives (Dreps)
+
+**Dedications:** For my daughter, Gwendolyn, so that the beautiful things of this world do not fall to darkness. And for my son, Theodore, to prove to you there is no great evil in this world that can not be defeated with an honest heart, true intent, and the courage to stand for what is right.
+
+## Motivation
+The proposal tabled by IOG is *deeply* flawed. The proposal effectively creates a slow-moving pseudo-decentralized system of soft-tyrannical top-down governance. 
+
+Their proposed system of governance makes it incredibly difficult for the community to replace the Constitutional Committee if it is not performing well or is not acting in the best interest of the network. This is because the process for replacing the committee is arbritrarily complex. Additionally, the size and quorum requirements of the committee can change each time it is replaced, which makes it difficult for the community to predict or understand how the committee operates. It also can be difficult to understand how the action will be ratified, and which actions will be enacted or dropped.
+
+Let us apply Occam's Razor, put simply “the simplest solution is almost always the best". A system of arbitrary complexity is often developed those of great intelligence - this issue plagues particle physics where increasingly complex models are proposed as a refutation to the standard model, requiring increasingly complex experimentation which ultimately results in increasingly expensive refutation of the underlying theory. This problem manifests in almost all fields of academia. 
```

Comment:
The commentary on particle physics and academia is, while interesting and sufficiently dramatic, probably superfluous and doesn't really contribute to an understanding of the mechanisms or motivation for your proposal, in my eyes.

Reply:
Hey @Quantumplation, thank you again for your feedback. As I have mentioned in my previous reply my schedule has been incredibly challenging recently - This is due to change shortly and I anticipate a definitive 'v1' of 1776 being ready for critique and review by the 1st of next month.

It somewhat superfluous, it comes from a discussion with my father-in-law who is completing his doctorate in solar physics at Cambridge. It is unlikely to appear in the final iteration of the CIP.